README.md

```diff
@@ -193,6 +193,18 @@ After installing the dependencies, you can lint the code by calling:
 npm run lint
 ```
 
+## Localization
+
+This repo includes a dedicated branch for localization called... `localization`.  To add localized text, add or update the relevant `.ftl` file under `locales/en`.  Be sure to reference the [localization documentation](https://mozilla-l10n.github.io/documentation/localization/index.html) for best practices.  
```

Comment:
That repo contains a lot of documents, only a few useful to devs. I would point directly to the best practices doc.
```suggestion
This repo includes a dedicated branch for localization called... `localization`.  To add localized text, add or update the relevant `.ftl` file under `locales/en`.  Be sure to reference the [localization documentation](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html) for best practices.  
```